### PR TITLE
contributes to fixing bug 1159993 - fix fennec ADI channel

### PR DESCRIPTION
**Pull request by twobraids**
integrated the update sql as a final step in the fetch ADI from hive job

### CODE REVIEW

**Comment by lonnen** [comment]
r+ on the code and tests. Once we confirm this bit of SQL is the right one we can land it.

File: socorro/cron/jobs/fetch_adi_from_hive.py [hunk 1]
```diff
@@ -119,6 +120,13 @@
         build_channel
 """
 
+_FENNEC38_ADI_CHANNEL_CORRECTION_SQL = """
+    update raw_adi
+        set update_channel = 'beta'
+        where product_name = 'FennecAndroid'
+              and product_version = '38.0'
+              and build = '20150427090529'
+              and date > '2015-04-27';"""
```
Comment (rhelmer) [inline socorro/cron/jobs/fetch_adi_from_hive.py]: This lgtm.

**Comment by lonnen** [comment]
Ok, this time the mocked test failed for the same reason.

File: socorro/unittest/cron/jobs/test_fetch_adi_from_hive.py [hunk 2]
```diff
@@ -177,6 +187,16 @@ def return_test_data(fake):
                 'build_channel': 'release-cck-\\',
                 'product_guid': '{a-guid}',
                 'count': 2
+            }, {
+                'report_date': datetime.date(2015, 4, 28),
+                'product_name': 'FennecAndroid',
+                'product_os_platform': 'Ginko',
+                'product_os_version': '3.1415',
+                'product_version': '38',
```
Comment (rhelmer) [inline socorro/unittest/cron/jobs/test_fetch_adi_from_hive.py]: At least one test problem is here - this should be `38.0` not `38`.

Comment (rhelmer) [inline socorro/unittest/cron/jobs/test_fetch_adi_from_hive.py]: I think that's the only test problem, the output matches otherwise.

**Comment by rhelmer** [comment]
Replaced by PR #2767

**Pull request closed without merging**